labs-ios-starter/Application/Extensions/String + identifier.swift

```diff
@@ -24,7 +24,16 @@ enum TableViewIdentifier: String {
     case profileCell
 }
 
+enum CollectionViewIdentifier: String {
+    case notificationsCollectionViewCell
+    case topicsCollectionViewCell
+    case questionsCollectionViewCell
+    case addNewQuestionCell
+    case crudCollectionViewCell = "CRUDCollectionViewCell"
+}
+
 extension String {
+
     ///convenience method to retrieve Storyboard Identifiers
     /// - Parameter identifier: defined in `enum StoryboardIdentifier` in String + identifier.swift
     /// - Returns: the rawValue of the case with the first letter capitalized
```

Comment:
This whole file could also be done like this if you thought this was cleaner
`//extension String {
	struct SegueIdentifier {
		static let profileDetailViewController = "ProfileDetailViewController"
		static let surveyTabBarController = "SurveyTabBarController"
		static let profileNavigationController = "ProfileNavigationController"
	}
//}
SegueIdentifier.profileDetailViewController`

Reply:
Well... that formatted horribly but I think you've seen that snippet before